lib/interop.js

```diff
@@ -468,7 +468,6 @@ Interop.prototype.toUnifiedPlan = function(desc) {
                         // msid. Note that "recvonly" is a term only defined
                         // for m-lines.
                         uLine.msid = sources[ssrc].msid;
-                        uLine.direction = 'sendrecv';
```

Comment:
Are you trying to fix a specific issue? I'm asking because this reverts 519e2b8b1bca77ed15c3c1ee434f5422fface905 which was meant to fix an issue another with FF. The problem was that the uLine was being set to recvonly (because the bLine used as a prototype was recvonly) so FF wasn't streaming video.